### ISSUE TITLE
implement loglevels

### ISSUE THREAD

**Issue by nielsole** [nielsole/go_tile#6]
broken paths can be hard to fix otherwise

**Comment by hummeltech** [comment]
I was experimenting a little bit with `klog`, let me know if this might be something you'd be satisfied with.
https://github.com/hummeltech/go_tile/commit/aabab4ba8dc4651b431c0c156f6fda9b079a1ece

**Comment by nielsole** [comment]
I would not like to add an external dependency for this. We only have a handful of messages that need loglevels anyways, so maybe I will just wrap them in if statements if the stdlib is not up for it. :ugly:

**Comment by hummeltech** [comment]
Yeah, good point, how about something like this?
https://github.com/nielsole/go_tile/compare/master...hummeltech:LogLevels

**Comment by nielsole** [comment]
Ah, didn't see this comment. Yeah pretty much. This is what I came up with: https://github.com/nielsole/go_tile/pull/10/files

**Comment by nielsole** [comment]
I've given this some more thought and I reverted my PR. I like your approach better. For the following reasons:
* No need for Sprintf
* Fatal is more in line with existing stdlib than Critical and avoids confusion about whether or not the call will call os.Exit(1).
* No need for a struct. Making a struct and then making it a global kind of defeats the point. Might as well be flat function calls. A struct feels a bit more like idiomatic go and direct function calls feel a bit like C, but so what.
* I like your other changes, e.g. renaming from tls to https, cleaning up the mux creation, cleanup of duration flag parsing and they are not trivial to separate

Things I think could be improved:
* How would one disable verbose in the docker container (except overriding the entrypoint)? Probably just something you left in for testing.
* Should we add the timestamp similar to what I did in my PR?
